src/lib/api/UserAPI.ts

```diff
@@ -40,6 +42,25 @@ export const getCurrentUser = async (token: string): Promise<PrivateProfile> =>
   return response.data.user;
 };
 
+export const getCurrentUserAndRefresh = async (
```

Comment:
Can we name this into something like `getCurrentUserAndRefreshCookie`? Right now it's not obvious what 'refresh' means